visualization/app/codeCharta/util/fileValidator.ts

```diff
@@ -16,35 +16,51 @@ export interface CCValidationResult {
 }
 
 export class FileValidator {
-	private static FILE_IS_INVALID = ["file is empty or invalid", "Error Loading File"]
-	private static API_VERSION_IS_INVALID = ["file API Version is empty or invalid", "File API Version Error"]
-	private static API_VERSION_IS_OUTDATED = [
-		"API Version Outdated: Update CodeCharta API Version to match cc.json",
-		"Error CodeCharta Major API Version"
-	]
-	private static MINOR_API_VERSION_IS_OUTDATED = ["Minor API Version Outdated", "Warning CodeCharta Minor API Version"]
-	private static NODES_NOT_UNIQUE = ["node names in combination with node types are not unique", "Uniqueness Error"]
-	private static VALIDATION_ERROR_TITLE = "Validation Error"
+	private static ERROR_MESSAGES = {
+		fileIsInvalid: {
+			title: "Error Loading File",
+			message: "file is empty or invalid"
+		},
+		apiVersionIsInvalid: {
+			title: "Error File API Version",
+			message: "file API Version is empty or invalid"
+		},
+		apiVersionIsOutdated: {
+			title: "Error CodeCharta Major API Version",
+			message: "API Version Outdated: Update CodeCharta API Version to match cc.json"
+		},
+		minorApiVersionOutdated: {
+			title: "Warning CodeCharta Minor API Version",
+			message: "Minor API Version Outdated"
+		},
+		nodesNotUnique: {
+			title: "Error Name-Type Uniques",
```

Comment:
I think this is clearer for the user
```suggestion
			title: "Error Node Uniques",
```